sbt-bridge/src/xsbt/DelegatingReporter.java

```diff
@@ -88,7 +88,7 @@ public void doReport(MessageContainer cont, Context ctx) {
           else return Optional.ofNullable(src.file().path());
         }
         public Optional<Integer> line() {
-          int line = pos.line();
+          int line = pos.line() + 1;
```

Comment:
Do we need to do anything for columns? Either way a test would be welcome.

Reply:
From what I've seen the columns are correct. I will add a test for that.

Reply:
Done